package/cloudshell/iac/terraform/services/tf_proc_exec.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, shell_helper: ShellHelperObject, sb_data_handler: SandboxData
 
     def init_terraform(self):
         self._shell_helper.logger.info("Performing Terraform Init...")
-        self._shell_helper.sandbox_messages.write_message("Running Terraform Init...")
+        self._shell_helper.sandbox_messages.write_message("running Terraform Init...")
```

Comment:
Why lower case? Is that just our standard?

Reply:
Yes. All other similar messages are written with lower case